content/en/docs/instrumentation/php/automatic.md

```diff
@@ -57,6 +57,14 @@ function ends it.
       how to do that using a specific version of the extension (1.0.0beta2 in
       this case):
 
+      **Installing from source requires proper development environment and few
+      dependendencies:**
+
+      {{< ot-tabs "Linux (apt)" "MacOS (homebrew)">}}
+      {{< ot-tab lang="shell">}} $ sudo apt-get install gcc make autoconf {{< /ot-tab >}}
+      {{< ot-tab lang="shell">}} $ brew install gcc make autoconf {{< /ot-tab >}}
+      {{< /ot-tabs >}}
+
```

Comment:
```suggestion
With your environment setup you can install the extension:
```

The two code blocks look odd without being separated by a sentence, so here's a suggestion what to put inbetween